packages/gatsby-theme-api-docs/src/utils/extract-additional-info.js

```diff
@@ -43,9 +29,24 @@ const extractAdditionalInfo = (property) => {
       delete additionalInfo[key];
   });
 
-  additionalInfo = reorderFields(additionalInfo, ['default']);
+  const tagIdentifier = ['max', 'min', 'default'];
+  let sortedList = Object.entries(additionalInfo);
+
+  tagIdentifier.forEach((tag) => {
+    sortedList = sortedList.reduce((list, item) => {
+      if (item[0].includes(tag)) {
+        return [item, ...list];
```

Comment:
nitpick that using array push / pop is faster because you do not clone the complete array.  But we're dealing with really small amounts of data here so don't worry.